pkg/chains/formats/format.go

```diff
@@ -31,10 +31,12 @@ const (
 	PayloadTypeTekton        config.PayloadType = "tekton"
 	PayloadTypeSimpleSigning config.PayloadType = "simplesigning"
 	PayloadTypeInTotoIte6    config.PayloadType = "in-toto"
+	PayloadTypeSlsav1        config.PayloadType = "slsa/v1"
 )
 
 var (
-	payloaderMap = map[config.PayloadType]PayloaderInit{}
+	IntotoAttestationSet = map[config.PayloadType]struct{}{PayloadTypeInTotoIte6: {}, PayloadTypeSlsav1: {}}
```

Comment:
The fact we need to plumb this through is probably a sign that we need better interfaces for the format -> storage transformation. :(

My guess is that the code is too tied to storage opts that it would be a pain in the ass to switch to the `Type()` method on the format interface?

(I'll take a look at ways we can refactor this soon)